src/movies.js

```diff
@@ -2,14 +2,111 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(movies) {
+  const newArray = [...movies];
+  newArray.sort(function(movA, movB) {
+    if (movA.year < movB.year) {
+      return -1;
+    }
+    if (movA.year > movB.year) {
+      return 1;
+    }
+    if (movA.year === movB.year) {
+      if (movA.title < movB.title) {
+        return -1;
+      }
+      if (movA.title > movB.title) {
+        return 1;
+      }
+      if (movA.title === movB.title) {
+        return 0;
+      }
+      return 0;
```

Comment:
This return is redundant as you are already considering all possible cases and returning 0 when titles are the same. So you could remove it.